src/elfpy/simulators.py

```diff
@@ -376,10 +398,14 @@ def collect_and_execute_trades(self, last_block_in_sim=False):
                 agent.log_status_report()
                 self.update_analysis_dict()
                 self.run_trade_number += 1
+                number_of_executed_trades += 1
+        if number_of_executed_trades > 0:
+            logging.debug("executed %f trades at %s", number_of_executed_trades, self.market.get_market_state_string())
```

Comment:
This adds a debug print for every trade, which will generate a massive amount of outputs. Maybe we should introduce a level higher than debug to warn users that they're getting hella outputs? It seems like there would be good argument for a debug output that is still viable with more than a few days, blocks per day, and trades per block.